main.jsx

```diff
@@ -92,17 +56,18 @@ const Tag = router => {
 
 const Routes = () => (
   <Switch>
-    <Route exact path="/" render={() => <Redirect to="/featured" />} />
-    <Route path="/featured" component={Featured} />
+    <Route exact path="/" component={Search} />
+    <Route path="/search/:tab" render={(props) => <Redirect to={`/${props.match.params.tab}${props.location.search}`}/>}/>
```

Comment:
This feels a bit opaque- why is it a Route with a Redirect in the action, instead of a Redirect component right away? Something to do with passing parameters?

Reply:
Yeah, it strips out the query params.